### DISCUSSION TITLE
Get list of users within individual geographical range

### DISCUSSION THREAD

**Discussion by GauravCalidig** [supabase/supabase#8461] [General]
I am storing the lat long of users in the user table, Is there any way I can filter users within a radius in Kms.

**Comment by GauravCalidig** [comment]
@GaryAustin1  Any ideas regarding this?

**Comment by GaryAustin1** [comment]
I'm sure it can be done with an rpc call to a postgres function (or possibly a view).  You would need the PostGIS extension to do the filter (where).  I've not personally done that so can't suggest exact code.
I store the user's location in a PostGIS geometry column: 
```
alter table notes
    add location geometry(Point, 4326);
```
And can see them on my Datagrip IDE map feature, so I know they work.  Just have not done the code to search them yet.
I set them in the client (browser) with:
```
note.location = "SRID=4326;POINT(" + geo.coords.long + " " + geo.coords.lat + ")"
```
I'm sure you can google and find the code needed for the Postgres filter.

**Reply by gauravmehta13** [reply]
Thanks will check

**Comment by GauravCalidig** [comment]
this is also helpful https://github.com/dshukertjr/spot
